.github/CONTRIBUTING.md

```diff
@@ -50,13 +46,13 @@ These tests are run both with the latest available version of `Nextflow` and als
 
 :warning: Only in the unlikely and regretful event of a release happening with a bug.
 
-- On your own fork, make a new branch `patch` based on `upstream/master`.
+- On your own fork, make a new branch `patch` based on `upstream/main`.
 - Fix the bug, and bump version (X.Y.Z+1).
-- A PR should be made on `master` from patch to directly this particular bug.
+- A PR should be made on `main` from patch to directly this particular bug.
 
 ## Getting help
 
-For further information/help, please consult the [nf-core/staramr documentation](https://nf-co.re/staramr/usage) and don't hesitate to get in touch on the nf-core Slack [#staramr](https://nfcore.slack.com/channels/staramr) channel ([join our Slack here](https://nf-co.re/join/slack)).
+For further information/help, please consult the [nf-core/staramr documentation](https://nf-co.re/staramr/usage).
```

Comment:
Can you switch to reference this git repo (https://github.com/phac-nml/staramrnf).

Reply:
Will do!